Makefile

```diff
@@ -32,7 +31,7 @@ dev: node_modules/.uptodate
 .PHONY: clean
 clean:
 	rm -f node_modules/.uptodate
-	rm -rf build/* build/.settings.json build/.*.deps
```

Comment:
I removed all the logic related to building the `.deps` file because IMO it doesn't add a lot of value. We can just rebuild the bundle whenever any JS file in `src/` changes or whenever the yarn lockfile changes.